src/parse/binary/subnode-handler.ts

```
@@ -0,0 +1,135 @@
+/**
+ * FBX 子节点处理器
+ * 负责处理各种特殊类型的子节点
+ */
+
+import type { FBXConnectionNode, FBXTree, FBXTreeNode, FBXTypedProperty } from '../../types';
```

ESLint

[error] 'FBXConnectionNode' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)